db/migrate/20180407125848_add_additional_information_to_activities.rb

```diff
@@ -0,0 +1,5 @@
+class AddAdditionalInformationToActivities < ActiveRecord::Migration[5.0]
+  def change
+    add_column :activities, :additional_information, :jsonb, null: false, default: {}
```

Comment:
@lucaspinto FYI: I changed the `default` value here. In Rails 4.1 `default: '{}'` happened to work, but it was actually a bug: https://github.com/rails/rails/issues/25594. In Rails 5 `default: {}` should be used.